prod/westeurope/internal/api/functions_iopayportal/subnet/terragrunt.hcl

```diff
@@ -0,0 +1,35 @@
+# Common
+dependency "virtual_network" {
+  config_path = "../../../../common/virtual_network"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_subnet?ref=v2.1.0"
+}
+
+inputs = {
+  name = "fn3bonusapi"
```

Comment:
The subnet name here is duplicated ... and it's not following the naming convention: I am expecting something lilke *iopay*